quickstarts/nextcloud/config.yml

```diff
@@ -0,0 +1,39 @@
+slug: nextcloud
+
+description: |
+  ## Comprehensive monitoring quickstart for Nextcloud
+  With New Relic's Nextcloud monitoring system, keep an eye on your Nextcloud in real-time, correlate transactions moving across the application environment and troubleshoot the issue which you are facing.
+  ### Why monitor Nextcloud?
+  Control, monitor and secure all the data of Nextcloud with Infrastructure monitoring agent and Prometheus open metric integration to view a thorough Nextcloud server dashboard that contains a high-level overview of their app's performance. 
```

Comment:
"Use the power of our infrastructure monitoring agent and the Prometheus open metric integration to control, monitor, and secure the data on your Nextcloud server. Use our dashboards to get a high-level overview of your server's performance."